engine/src/main/java/org/terasology/rendering/dag/nodes/LightShaftsNode.java

```diff
@@ -146,7 +147,7 @@ public void process() {
 
         sunPositionWorldSpace4.set(sunDirection.x * 10000.0f, sunDirection.y * 10000.0f, sunDirection.z * 10000.0f, 1.0f);
         sunPositionScreenSpace.set(sunPositionWorldSpace4);
-        activeCamera.getViewProjectionMatrix().transform(sunPositionScreenSpace);
+        new Matrix4f(activeCamera.getViewProjectionMatrix()).transpose().transform(sunPositionScreenSpace);
```

Comment:
[Matrix4f.transformTranspose(Vector4f)](https://github.com/JOML-CI/JOML/commit/094f9fd49afec20eb3d46b03e16a411c9be2fe4e) will be available in JOML 1.9.25.

Reply:
that's nice to know about. I'll keep that in mind when we do another version bump.